### PR TITLE
fix bug:Lose size info when read from dehydrate string

### PR DESCRIPTION
**Pull request by pipi32167**

### CODE REVIEW

File: app/BitSet.js [hunk 1]
```diff
@@ -30,7 +30,11 @@ BitSet = function (nBitsOrKey) {
       for (i = 0; i < arrVals.length; i++) front[leadingZeros + i] = arrVals[i];
       arrVals = front;
     }
-    this.arr = new Uint32Array(arrVals);
+    wordCount = Math.ceil((this.MAX_BIT + 1) / BITS_PER_INT);
+    this.arr = new Uint32Array(wordCount)
+    for (var i = 0; i < arrVals.length; i++) {
```
Comment (mattkrick) [inline app/BitSet.js]: replacing the loop with `this.arr.set(arrVals)` should be faster

**Comment by mattkrick** [comment]
Thanks for the PR! LGTM after we replace the loop with the `set` method.

File: app/BitSet.js (hunk 1 above)
Comment (pipi32167) [inline app/BitSet.js]: Yep, u r right, should I replace the loop and PR again?

Comment (mattkrick) [inline app/BitSet.js]: Sure, no need for a squash or anything, you should be able to patch it
straight from github.

**Comment by pipi32167** [comment]
Please accept the PR.

**Pull request merged**